libs/coin-evm/src/prepareTransaction.ts

```diff
@@ -1,13 +1,13 @@
-import { isEqual } from "lodash";
-import BigNumber from "bignumber.js";
-import { Account, TokenAccount } from "@ledgerhq/types-live";
 import { findSubAccountById } from "@ledgerhq/coin-framework/account/index";
-import { EvmNftTransaction, Transaction as EvmTransaction } from "./types";
-import { getTransactionData, getTypedTransaction } from "./transaction";
-import { validateRecipient } from "./getTransactionStatus";
+import { Account, TokenAccount } from "@ledgerhq/types-live";
+import BigNumber from "bignumber.js";
+import { isEqual } from "lodash";
 import { getNftCollectionMetadata } from "./api/nft";
 import { getFeesEstimation, getGasEstimation, getTransactionCount } from "./api/rpc";
+import { validateRecipient } from "./getTransactionStatus";
 import { getAdditionalLayer2Fees, getEstimatedFees, isNftTransaction } from "./logic";
+import { getTransactionData, getTypedTransaction } from "./transaction";
+import { EvmNftTransaction, Transaction as EvmTransaction, FeeData } from "./types";
 
```

Comment:
My beautiful pyramidal order 😢

Reply:
Indeed, but VSCode automation and shortcuts https://code.visualstudio.com/docs/typescript/typescript-refactoring#_organize-imports feels much better and powerful than manual imports ordering 👨‍💻

Reply:
But it's not... PYRAMIDAL. 📐🎨